content/backmatter.tex

```diff
@@ -667,9 +667,11 @@ \chapter{Changes to this Document}\label{sec:changelog}
 
 \section{Version 1.6}
 \label{changelog:v1.6}
-Major changes in \openshmem[1.6] include the addition of the new
-\FUNC{shmem\_team\_ptr}, \FUNC{shmem\_ibget}, and \FUNC{shmem\_ibput}
-functions.
+Major changes in \openshmem[1.6] include the addition of inclusive and
+exclusive scan operations, session routines, support for multiple
+initialization and finalization calls, interleaved block transfer APIs,
+signal add and set APIs, and the
+\FUNC{shmem\_team\_ptr} function.
```

Comment:
Perhaps `shmem_team_ptr` is not a "major change"?  It's at least included below...

Reply:
It was already there, so I'd rather not remove it at this stage..